pkg/proxy/util/utils.go

```diff
@@ -223,3 +229,13 @@ func IsVIPMode(ing v1.LoadBalancerIngress) bool {
 	}
 	return *ing.IPMode == v1.LoadBalancerIPModeVIP
 }
+
+func RetryOnIntr(f func() error) {
+	for attempt := 0; attempt < MaxAttemptsEINTR; attempt += 1 {
+		if err := f(); !errors.Is(err, unix.EINTR) {
+			return
+		}
+	}
+	klog.V(3).InfoS("netlink call interrupted", "attempts", MaxAttemptsEINTR)
```

Comment:
```suggestion
	klog.V(2).InfoS("netlink call interrupted", "attempts", MaxAttemptsEINTR)
```

Reply:
Fixed in f9fcbaecbe02b08961c4caa496768eb0c6646346